src/identity-utils.ts

```diff
@@ -279,3 +280,28 @@ const getExpireTimestamp = (maxAge: number = ONE_DAY_IN_SECONDS): number =>
 
 const parseIdentityResponse = (responseText: string): IdentityResultBody =>
     responseText ? JSON.parse(responseText) : ({} as IdentityResultBody);
+
+export const hasIdentityRequestChanged = (
+    mpInstance: IMParticleWebSDKInstance
+): boolean => {
+    const currentUser = mpInstance.Identity.getCurrentUser();
+
+    const newIdentityRequest = mpInstance._Store.SDKConfig.identifyRequest;
+
+    if (
+        !currentUser ||
+        !newIdentityRequest ||
+        !newIdentityRequest.userIdentities
+    ) {
+        return false;
+    }
```

Comment:
```suggestion
    if (!currentUser || !newIdentityRequest?.userIdentities) {
        return false;
    }
```

or if you really wanna make it a one liner:

`return !!(currentUser && newIdentityRequest?.userIdentities);`